nncf/experimental/openvino_native/quantization/quantizer_parameters.py

```diff
@@ -204,24 +204,7 @@ def calculate_weight_quantizer_parameters(weight_tensor: np.ndarray, quantizer_c
     :return: Parameters of the FakeQuantize layer.
     """
     quant_group = QuantizerGroup.WEIGHTS
-    if quantizer_config.per_channel:
-        bounds_shape = get_weight_stats_shape(weight_tensor.shape, metatype)
-        axes = tuple(i for i, dim in enumerate(bounds_shape) if dim == 1)
-    else:
-        axes = None
-
-    max_values = np.amax(np.abs(weight_tensor), axis=axes, keepdims=quantizer_config.per_channel)
```

Comment:
Where did this logic migrate to?

Reply:
Now max/min value is being collected by `OVMinMaxStatisticCollector`s